### PR TITLE
Alert or success message on newsletter subscription form

### PR DESCRIPTION
**Pull request by ghost**
Falta ponerlo bonito, ¿alguien que tenga mano con CSS?

### CODE REVIEW

File: pycones/web/static/web/js/web.js [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
+                                .delay(3000)
+                                .fadeOut();
 
+      };
+
+      var form = $("#newsletter-form");
+
+      $.post(form.attr('action'), form.serialize(), null, 'json')
+       .done(function(data){
+         showMessage(data.message, "alert alert-success");
```
Comment (niwinz) [inline pycones/web/static/web/js/web.js]: Why two space indent? Are you kidding me?

Comment (ghost) [inline pycones/web/static/web/js/web.js]: I tend to put all the chained calls at the same level of indentation.

Comment (niwinz) [inline pycones/web/static/web/js/web.js]: Yeah, but showMessage function call is not a chained call :P

**Comment by niwinz** [comment]
I like it now! Thanks!

**Comment by ghost** [comment]
Yeah, much better. The devil is in the details!

**Comment by yamila-moreno** [comment]
Mergeado manualmente.

**Pull request closed without merging**